src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -511,6 +511,9 @@
   <data name="OpenWindowRenamerCommandKey" xml:space="preserve">
     <value>Rename window...</value>
   </data>
+  <data name="DisplayWorkingDirectoryCommandKey" xml:space="preserve">
+    <value>Display Terminal's current working directory</value>

```

Comment:
This is too nuanced I think for users to get - it's a debugging feature, and it will **NOT** shoe the current working directory of the shell that is hosted in the current tab inside the Terminal. I might keep this one on the drawing board for 18

Reply:
@DHowett What if I put this behind `DebugFeaturesEnabled`, like the "Break into debugger" action? At least in that case, "you must be this high to ride" applies

Reply:
alternatively, name the action `debugTerminalCwd`

Reply:
How about both? In this case, I will consider `debug` to be the verb :grin: